### ISSUE TITLE
Vieb 6.0.0 crashes on quitall on macOS 11.6 (M1)

### ISSUE THREAD

**Issue by al3xandru** [Jelmerro/Vieb#317]
**Describe the bug**

Every time I quit Vieb using quiteall, the app crashes. I can provide the crash report if needed. 

**Specs**
 - OS: macOS 
 - Version: 11.6
 - Apple Silicon

**Comment by Jelmerro** [comment]
I don't think this is specifically a Vieb issue, do you have the same problem with other electron based apps? I can see a bunch of issues in the Electron issue tracker with a similar description: [see here](https://github.com/electron/electron/issues?q=label%3Aplatform%2FmacOS+label%3A"crash+%3Aboom%3A")

**Comment by al3xandru** [comment]
I don't have other electron based apps except Slack, but doesn't mean that this is not related to Electron itself.

**Comment by Jelmerro** [comment]
Does the stack trace and/or crash report look similar to [this one](https://github.com/electron/electron/issues/30786)?

**Comment by Raghav-Dixit** [comment]
I think it uses some weird signal like sigquit or sigkill when you run `:quitall`. There is nothing wrong. MacOS notifies you when you "force an application to quit". It thinks the app crashed when it gets the `sigquit` or `sigkill` signals. It's the same effect as running force quit on an application. I think it might be fixable by using `sigterm`, which respectfully "asks" the process to stop. Run `man 7 signal` or `man signal` in the terminal to find out more about signals.

**Comment by Jelmerro** [comment]
Please see the [linked issue](https://github.com/electron/electron/issues/30786), this does not seem to be a Vieb issue specifically, moreover an Electron issue.

**Comment by Jelmerro** [comment]
I recently have been able to test this on an M1 macbook, and have not been able to reproduce this. Is this still an issue?

**Comment by Jelmerro** [comment]
I really cannot reproduce this at all with the latest Vieb and macos, so closing for now. Feel free to let me know the macos and Vieb version this still affects if any.
